TheWarWithin/RogueSubtlety.lua

```diff
@@ -933,15 +948,15 @@ spec:RegisterAbilities( {
         handler = function ()
             removeBuff( "masterful_finish" )
 
-            if talent.alacrity.enabled and effective_combo_points > 4 then addStack( "alacrity" ) end
+            if talent.alacrity.enabled and effective_combo_points > 9 then addStack( "alacrity" ) end
```

Comment:
I think this is fine to stay at `> 4` -- at 5+ combo points, Alacrity will add a stack.  Is there some other effect to be accounted for here?

![image](https://github.com/user-attachments/assets/48565914-4e57-4592-adba-43b3caa3b875)

Reply:
![image](https://github.com/user-attachments/assets/007269e8-0b41-4e3b-8173-bf43dcb5b5ae)

 ??